core/server/app.js

```diff
@@ -22,12 +23,18 @@ module.exports = function setupParentApp() {
     // (X-Forwarded-Proto header will be checked, if present)
     parentApp.enable('trust proxy');
 
+    parentApp.use(responseTime());
```

Comment:
Can we please (please, please) use the same request logging middleware we already have in our internal code?

This `responseTime` dependency adds a header, which is very different to the intention of the TODO, which was to fix https://github.com/TryGhost/Ghost/pull/7597/commits/0541af87ad81176d9ba3d77c793a3281cd0db02f#diff-85567d8f2026caaf1a67588fb1641aa9R32 which is IMO too simplistic and to also add duration to the logs.

The middleware I'm suggesting we use:
- adds duration without needing a dependency
- also adds an id for requests, so multiple logs from the same request are traceable
    - note this requires a dependency for generating a uuid, but we can use the same one we will use for creating a UID for errors and could even use the uid util that already exists
- has more advanced handling of request/responses borrowed from morgan, which covers more edge-cases for the behaviour of requests and responses.